interfaces/builtin/kubernetes_support.go

```diff
@@ -108,6 +108,11 @@ profile systemd_run (attach_disconnected,mediate_deleted) {
   @{INSTALL_DIR}/{@{SNAP_NAME},@{SNAP_INSTANCE_NAME}}/@{SNAP_REVISION}/{,usr/}bin/true ixr,
 ###KUBERNETES_SUPPORT_SYSTEMD_RUN###
 }
+
+# Lock file used by Calico's IPAM plugin. This is configurable via the
+# (undocumented) "ipam_lock_file" configuration key:
+# https://github.com/projectcalico/cni-plugin/blob/master/pkg/types/types.go
+/{,var/}run/calico/ipam.lock rwk,
```

Comment:
actually though, could you clarify if this access for calico is needed for kubelet, kubeproxy, or both? If both then this is okay in this variable, but if it's only one of these then it should be moved to one of the other flavor variables

Also, looking at where this is put, I don't like this here as it is difficult to find when reading the full file/profile, as this is below the systemd_run attach rule, but this is not meant to be part of the systemd_run profile, it should be part of the k8s profile itself, so if this could be moved above the systemd_run bits that would be great

Reply:
so if you could just move the definition up above the systemd_run bit that would be great